context.go

```diff
@@ -85,6 +85,10 @@ func (context *Context) DialConfig(config *websocket.Config) (*Sender, error) {
 	}
 	sender := context.start(ws)
 	go func() {
+		defer decrReceiverGoroutines()
+		defer sender.Stop()
```

Comment:
Add a comment to clarify the intended defer behaviour.

Reply:
Yeah, the incr and decr functions were mysterious to me until I finally found them in the expvar source file. Also, it seems more natural to me to put the `defer decr...` call right _after_ the `incr` call, instead of before it.